contributing-guides/maintainers-guide.md

```diff
@@ -145,3 +147,55 @@ for the behavior expected of tldr-pages maintainers.
 - If the CLA check is frozen at the message "Status waiting to be reported", it is recommended to close and reopen the pull requests to retrigger the check (and notify the contributor about the same).
 
 For reference to see if a contributor has signed the CLA, visit the dashboard at <https://cla-assistant.io/>.
+
+## V. Creating a client specification release
+
+### Pre-requisites
+
+- Ensure client specification changes are discussed with the other maintainers and community members in GitHub and chatroom, and the changes have been agreed upon and enough time has been provided for everyone to review the changes.
+- Tag all client spec PRs under a [milestone](https://github.com/tldr-pages/tldr/milestones) for ease of release.
+- Ensure [GPG signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key) has been setup for your account.
+- It is suggested to prepare the release notes to add to the client specification release in advance.
+    - The release notes should mention pending breaking architectural changes from previous client specifications (if any).
+    - Along with the changelog, the release notes must explain the client specification changes in detail along with examples (if any).
+
+### Steps
+
+1. Merge all applicable PRs that modify the client specification and ensure they are documented in the [CHANGELOG section of client specification](/CLIENT-SPECIFICATION.md#changelog).
+2. Bump the version to the upcoming release (in the client specification file) and inform other maintainers in the chatroom about the release.
+3. Clone the repository locally to your device: 
+
+```sh
+git clone https://github.com/tldr-pages/tldr
+```
+
+4. Cross check the additions, version and changelog details in the client specification file.
+5. Create a signed tag using the command 
+
+```sh
+git tag -s vX.Y.Z
+```
+
+> [!NOTE]
+> Replace `X.Y.Z` with the client specification version.
+
+6. Verify the created signed tag's details using the command: 
+
+```sh
+git tag -v vX.Y.Z
+```
+
+7. Now, push the tag to the repository using the command: 
+
+```sh
+git push origin vX.Y.Z
+```
+
```

Comment:
This clashes with the maintainers guide that contribution process should be kept web-based https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#i-responding-to-contributions

Tags seem to be createable in the releases menu https://stackoverflow.com/questions/18216991/create-a-tag-in-a-github-repository

Reply:
> This clashes with the maintainers guide that contribution process should be kept web-based https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#i-responding-to-contributions
> 
> Tags seem to be createable in the releases menu https://stackoverflow.com/questions/18216991/create-a-tag-in-a-github-repository

That's fine since creating releases isn't part of the normal contribution process of a majority of contributors and web based tags don't allow signing it.